scripts/build/Jenkins/Jenkinsfile

```
@@ -624,10 +624,15 @@ def PreBuildCommonSteps(Map pipelineConfig, Map params, String snapshot, String
             if(env.PROJECT_OVERRIDE) {
                 external_project = env.PROJECT_OVERRIDE
             }
-            else if(params.CMAKE_LY_PROJECTS) {
+            else if(params && params.CMAKE_LY_PROJECTS) {
                 external_project = params.CMAKE_LY_PROJECTS
             }
-
+            else {
+                echo "No PROJECT_OVERIDE or CMAKE_LY_PROJECTS  defined"
```

typo: `PROJECT_OVERIDE`
Also there is an extra space before the work "defined"

```suggestion
                echo "No PROJECT_OVERRIDE or CMAKE_LY_PROJECTS defined"
```